platform/src/main/java/org/stellar/anchor/platform/custody/CustodyPaymentHandler.java

```diff
@@ -89,9 +119,6 @@ protected void validatePayment(JdbcCustodyTransaction txn, CustodyPayment paymen
           payment.getAssetType(),
           payment.getId(),
           payment.getExternalTxId());
-      payment.setStatus(CustodyPaymentStatus.ERROR);
```

Comment:
I think, that we can go without this validation right now. We will update it, if it is needed. Check whether we can remove `message` and `status` from `CustodyPayment`

Reply:
Status is still used.